src/api/movies-api.ts

```diff
@@ -0,0 +1,26 @@
+import Axios, { AxiosResponse } from 'axios';
+import { MovieEntity } from './model';
+
+const serverURL = 'localhost:3000';
+const getMembersURL = `${serverURL}/api/films`;
```

Comment:
I would move this to a common place at least the base serverURL.

on the other hand I'm going to open a case to store this en environment variables